### PR TITLE
Weav 192 Query not equal/less than/...

### PR DESCRIPTION
**Pull request by mathieub93**
Matching PR in SDK-js

### CODE REVIEW

**Comment by malamili** [comment]
Looks quite nice! I think Gijs will need to take a deeper look as I dont fully grasp this connector thing yet 👍

File: src/main/resources/templates/nodes-new.fsql [hunk 1]
```diff
@@ -43,6 +44,40 @@ WITH
         </#list>
         )
         </#if>
+
+
+        <#list query.getRelationArrayConditions()?keys as conditionName>
```
Comment (gvdent) [inline src/main/resources/templates/nodes-new.fsql]: This block seems unrelated to the rest of the PR, I expect a merge artifact. Try removing it and see whether the tests still pass.

File: src/main/java/com/weaverplatform/postgresql/database/query/model/WeaverQuery.java [hunk 2]
```diff
@@ -565,6 +587,25 @@ public boolean hasExclusiveIDRestrictions() {
         }
         result.add(clause);
       }
+
+      if(condition.containsKey("$gt") || condition.containsKey("$gte") || condition.containsKey("$lt") || condition.containsKey("$lte") || condition.containsKey("$ne")) {
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/postgresql/database/query/model/WeaverQuery.java]: Nice 👍 

There is one other use case for these comparators: datetime_value's (they should be added as OR clauses on the same level as the double_value comparisons).

File: src/main/resources/templates/nodes-new.fsql (hunk 1 above)
Comment (mathieub93) [reply src/main/resources/templates/nodes-new.fsql]: Removed it, nothing broke.
*about to add new PR.*
